### ISSUE TITLE
PresentationDetents and DragIndicatorVisibility are not applied after rotating the device

### ISSUE THREAD

**Issue by mwocu** [shaps80/SwiftUIBackports#36]
*Have you read the [Contributing Guidelines](https://github.com/shaps80/.github/blob/master/CONTRIBUTING.md)?*

Yes

## General Information

 - Project version:
v1.9.0 (latest release)
 
 - Platform/OS version: 
iOS 16 (Simulator, any device)
 
 - IDE version:
Xcode 14.2

## Describe the bug
After rotating the device from portrait to landscape and back, the PresentationDetents and the visibility of the DragIndicator are not applied correctly anymore.

## Minimal reproduction
```
struct SheetView: View {
    @State private var showSheet = false
    
    var body: some View {
        Button("Show Sheet") {
            showSheet = true
        }
        .sheet(isPresented: $showSheet) {
            Color.red.edgesIgnoringSafeArea(.all)
                .backport.presentationDetents([.medium])
                .backport.presentationDragIndicator(.visible)
        }
    }
}
```
## Steps to reproduce
1. Use the example code from above
2. Start app in portrait mode
3. Rotate to landscape and back to portrait
4. The sheet is now showing in `.large` mode and the drag indicator is hidden

## Expected behavior
The sheet should still show up in `.medium` mode and the drag indicator should still be visible

## Bugfix branch
I already fixed the bug in this branch https://github.com/mwocu/SwiftUIBackports/tree/fix_sheet_presentation_after_device_rotation
and could submit a PR, if wanted
https://github.com/shaps80/SwiftUIBackports/compare/main...mwocu:SwiftUIBackports:fix_sheet_presentation_after_device_rotation

**Comment by shaps80** [comment]
would love this in a PR 👍🏻

**Comment by shaps80** [comment]
@mwocu this has now been released in 1.9.1
